src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -441,6 +441,16 @@ def _check_pid_from_log_file(self, pid, abort_on_warning,
 
         for line in self._li.new_iter(pid=pid, stateful=True):
             self.save_log_line(line)
+            try:
+                msg = ''.join(line._fields[2:])
+                # Warn if a line references the name of the function it was in,
+                # but skip short function names or _internal suffixes.
+                if line.function in msg and len(line.function) > 6 and \
+                   not '{}_internal'.format(line.function) in msg:
```

Comment:
(pylint-unneeded-not) Consider changing "not '{}_internal'.format(line.function) in msg" to "'{}_internal'.format(line.function) not in msg"